### DISCUSSION TITLE
Use of mutation testing in rails_event_store

### DISCUSSION THREAD

**Discussion by belene** [RailsEventStore/rails_event_store#1471] [General]
Hello there!

My name is Ana. I noted that you use the mutation testing tool mutant in the project. 
I am a postdoctoral researcher at the University of Seville (Spain), and my colleagues and I are studying how mutation testing tools are used in practice. With this aim in mind, we have analysed over 3,500 public GitHub repositories using mutation testing tools, including yours! This work has recently been published in a journal paper available at https://link.springer.com/content/pdf/10.1007/s10664-022-10177-8.pdf. 

To complete this study, we are asking for your help to understand better how mutation testing is used in practice, please! We would be extremely grateful if you could contribute to this study by answering a brief survey of 21 simple questions (no more than 6 minutes). This is the link to the questionnaire https://forms.gle/FvXNrimWAsJYC1zB9.

Drop me an e-mail if you have any questions or comments (anasan@us.es). Thank you very much in advance!!

**Comment by mostlyobvious** [comment]
Thank you for letting us know! I've responded already 🙂
